services/core-web/src/components/Forms/incidents/IncidentForm.tsx

```diff
@@ -164,7 +163,7 @@ export const IncidentForm: FC<IncidentFormProps> = (props) => {
           />
           <br />
           <IncidentFormInternalDocumentComments
-            documents={documents}
+            documents={formValues?.documents}
```

Comment:
I think we might want to null coalesce these to []- looks like both `IncidentFormDocuments` and `IncidentFormInternalDocumentComments` does a `documents.filter` which will probably blow up with undefined